checks/cloud/aws/rds/no_classic_resources.cf.go

```diff
@@ -2,23 +2,11 @@ package rds
 
 var cloudFormationNoClassicResourcesGoodExamples = []string{
 	`---
-AWSTemplateFormatVersion: 2010-09-09
-Description: Good example of rds sgr
-Resources:
-
 `,
 }
 
 var cloudFormationNoClassicResourcesBadExamples = []string{
 	`---
-AWSTemplateFormatVersion: 2010-09-09
-Description: Bad example of rds sgr
-Resources:
-  Queue:
-    Type: AWS::RDS::DBSecurityGroup
-    Properties:
-      Description: ""
-
```

Comment:
ditto

Reply:
Yes. This check is related to resources using EC2-Classic, so the example provided is not relevant. Additionally, [AWS has completely retired EC2-Classic](https://aws.amazon.com/blogs/aws/ec2-classic-is-retiring-heres-how-to-prepare/) as of August 23, 2023, so this check should probably be retired altogether.